applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_reader.py

```diff
@@ -0,0 +1,98 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Geiser Armin, https://github.com/armingeiser
+#
+# ==============================================================================
+
+class Shape:
+    def __init__(self):
+        self.name = None
+        self.nodes = []
+        self.triangles = []
+
+
+def read_nodes(line, file):
+    nodes = []
+    while True:
+        if "[" in line:
+            break
+
+    line = next(file)
+
+    while True:
```

Comment:
This is dangerous and prone to be stuck in an infinite loop. Is there maybe a reasonable limit for this loop?